bug: Helm chart is missing a ClusterRole

# Bug Report

**Amazing GitHub Template version:** [v0.0.1](https://github.com/OrKarstoft/kscale/releases/tag/v0.0.1)

<!-- Please specify commit or tag version. -->

**Current behavior:**
When trying to scale up a namespace the kscale pod will crash with the below error.

```
panic: namespaces "default" is forbidden: User "system:serviceaccount:nightscaler:kscale" cannot get resource "namespaces" in API group "" in the namespace "default"

goroutine 127 [running]:
github.com/orkarstoft/kscale/pkg/k8s.ScaleNamespaceUp({0xc00034ec48, 0x7}, 0x34630b8a000)
	github.com/orkarstoft/kscale/pkg/k8s/ScaleNamespaceUp.go:28 +0x429
github.com/orkarstoft/kscale/pkg/pubsub.Listen.func1({0xc0000cfaa0?, 0x0?}, 0xc00012b810)
	github.com/orkarstoft/kscale/pkg/pubsub/Listen.go:52 +0x293
cloud.google.com/go/pubsub.(*Subscription).Receive.func2.2({0x1694fa0?, 0xc00012b810?})
	cloud.google.com/go/pubsub@v1.30.0/subscription.go:1184 +0x87
cloud.google.com/go/pubsub/internal/scheduler.(*ReceiveScheduler).Add.func1()
	cloud.google.com/go/pubsub@v1.30.0/internal/scheduler/receive_scheduler.go:84 +0x2e
created by cloud.google.com/go/pubsub/internal/scheduler.(*ReceiveScheduler).Add
	cloud.google.com/go/pubsub@v1.30.0/internal/scheduler/receive_scheduler.go:82 +0x352
```

<!-- Describe how the bug manifests. -->

**Expected behavior:**
That is doesn't crash and actually scale up the namespace

<!-- Describe what the behavior would be without the bug. -->

**Steps to reproduce:**
1. Deploy kscale using the helm chart.
2. Try to scale up a cluster.
3. See the world burn.

<!--  Please explain the steps required to duplicate the issue, especially if you are able to provide a sample application. -->

**Other information:**
There is just missing a ClusterRole and ClusterRoleBinding


<!-- List any other information that is relevant to your issue. Related issues, suggestions on how to fix, Stack Overflow links, forum links, etc. -->

--- a/chart/templates/serviceaccount.yaml
+++ b/chart/templates/serviceaccount.yaml
@@ -17,7 +17,7 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["namespaces"]
-  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  verbs: ["get", "list", "watch", "update", "patch"]
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding

--- a/chart/templates/serviceaccount.yaml
+++ b/chart/templates/serviceaccount.yaml
@@ -9,4 +9,26 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "chart.serviceAccountName" . }}
+rules:
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "chart.serviceAccountName" . }}
+subjects:
+- kind: ServiceAccount
+  name: {{ include "chart.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ include "chart.serviceAccountName" . }}
+  apiGroup: rbac.authorization.k8s.io
 {{- end }}

